doc/manual/src/hacking.md

```diff
@@ -1,19 +1,13 @@
-Hacking
-=======
+# Hacking
 
 This section provides some notes on how to hack on Hydra. To get the
 latest version of Hydra from GitHub:
 
     $ git clone git://github.com/NixOS/hydra.git
     $ cd hydra
 
-To build it and its dependencies:
-
-    $ nix-build release.nix -A build.x86_64-linux
-
-To build all dependencies and start a shell in which all environment
-variables (such as PERL5LIB) are set up so that those dependencies can
-be found:
+To enter a shell in which all environment variables (such as PERL5LIB)
```

Comment:
```suggestion
To enter a shell in which all environment variables (such as ``PERL5LIB``)
```

I tend to add code blocks to all paths and envs to make it easier to spot them. Maybe something to adapt.

Reply:
Thanks, I usually do. This document was ported from XML and has some clean-up along these lines to be done :). Fixed!